databricks_cli/pipelines/cli.py

```diff
@@ -134,7 +134,7 @@ def deploy_cli(api_client, spec_arg, spec, allow_duplicate_names, pipeline_id):
 @provide_api_client
 def delete_cli(api_client, pipeline_id):
     """
-    Stops a delta pipeline and deletes its associated Databricks resources. The pipeline can be
+    Stops the pipeline and deletes its associated Databricks resources. The pipeline can be
```

Comment:
I think we should say: "Stops the pipeline by cancelling any active update". It is not true anymore that 
- this deletes the associated resources (e..g, the maintenance jobs stays around, and we keep the cluster alive for a while)
- you don't need to redeploy to restart the pipeline. 
So let's just be brief here and say only the bare minimum.

Reply:
Updated, this message is same as `stop` command, behavior to user is the same in both the cases?